README.md

```diff
@@ -47,7 +49,8 @@ You can even use your own configs by mounting them over the default ones in `/op
 docker run --pid=host -v /etc:/etc:ro -v /var:/var:ro -t -v path/to/my-config.yaml:/opt/kube-bench/cfg/config.yaml aquasec/kube-bench:latest [master|node]
 ```
 
-> Note: the tests require either the kubelet or kubectl binary in the path in order to know the Kubernetes version. You can pass `-v $(which kubectl):/usr/bin/kubectl` to the above invocations to resolve this.
+> Note: the tests require either the kubelet or kubectl binary in the path in order to know the Kubernetes 
```

Comment:
Looks like an Incomplete sentence. I also think we could re-word like below to simplify and clarify:
```
Note: the tests require either the kubelet or kubectl binary in your PATH to *auto-detect the kubernetes version*
```

Reply:
Good catch. I also just found an un-closed bracket :-)